src/main/java/frc/robot/subsystems/Climber_Subsystem.java

```diff
@@ -87,73 +165,49 @@ public double getPitch() {
+        int horizontalPosition = horizontalR.getSensorCollection().getQuadraturePosition();
+        int correctedPosition = horizontalPosition - oldRHorizontal;
+        SmartDashboard.putNumber("Raw Right Horizontal Position", horizontalPosition);
+        SmartDashboard.putNumber("Right Horizontal Position", correctedPosition);
+        return correctedPosition;
     }
 
     public int getLHorizontalPos() {
-        return (-horizontalL.getSensorCollection().getQuadraturePosition()) - oldLHorizontal;
+        int horizontalPosition = horizontalL.getSensorCollection().getQuadraturePosition();
+        int correctedPosition = horizontalPosition - oldLHorizontal;
+        SmartDashboard.putNumber("Raw Left Horizontal Position", horizontalPosition);
+        SmartDashboard.putNumber("Left Horizontal Position", correctedPosition);
+        return correctedPosition;
     }
 
+
+    //Called every time the operator hits Y
     public void climberInit() {
-        // while (getHorizontalLLimit() != true) {
-        // horizontalL.set(.25);
-        // }
-        oldLHorizontal = getLHorizontalPos();
-        // while (getHorizontalRLimit() != true) {
-        // horizontalR.set(.25);
-        // }
-        oldRHorizontal = getRHorizontalPos();
-        // while (getVerticalLLimit() != true){
-        // verticalL.set(.25);
-        // }
-        oldLVertical = getLVerticalPos();
-        // while (getVerticalRLimit() != true){
-        // verticalR.set(.25);
-        // }
-        oldRVertical = getRVerticalPos();
-
-        verticalR.configForwardSoftLimitThreshold(oldRVertical - 1200);
-        verticalR.configReverseSoftLimitThreshold(oldRVertical - 21000);
-        verticalL.configForwardSoftLimitThreshold(oldLVertical - 1200);
-        verticalL.configReverseSoftLimitThreshold(oldLVertical - 21000);
-        horizontalR.configForwardSoftLimitThreshold(oldRHorizontal - 1200);
-        horizontalR.configReverseSoftLimitThreshold(oldRHorizontal - 21000);
-        horizontalL.configForwardSoftLimitThreshold(oldLHorizontal - 1200);
-        horizontalL.configReverseSoftLimitThreshold(oldLHorizontal - 21000);
 
-        verticalR.configForwardSoftLimitEnable(true);
-        verticalR.configReverseSoftLimitEnable(true);
-        verticalL.configForwardSoftLimitEnable(true);
-        verticalL.configReverseSoftLimitEnable(true);
-        horizontalL.configForwardSoftLimitEnable(true);
-        horizontalL.configReverseSoftLimitEnable(true);
-        horizontalR.configForwardSoftLimitEnable(true);
-        horizontalR.configReverseSoftLimitEnable(true);
+        //soft reset the encoders so that the PID reads 0 difference between them at the start.
+        //strictly speaking we probably don't need this for the vertical since we balance with the gyro
+        //we'll reset it for consistency though
+        oldLHorizontal = horizontalL.getSensorCollection().getQuadraturePosition();
```

Comment:
This is the thing that gets called every time we enter climber mode. We want the soft reset each time so that the robot stays straight when moving horizontally relative to its position when we entered climber mode. If we did this at startup instead when we started climbing the robot would try to correct for any bumps that the climber took during the match while we were in the air. That's no good